lib/truffle/rbconfig.rb

```diff
@@ -112,12 +112,21 @@ module RbConfig
   ruby_home = Truffle::Boot.ruby_home
 
   if ruby_home
-    prefix = ruby_home
-    bindir = "#{prefix}/bin"
+    prefix        = ruby_home
+    bindir        = "#{prefix}/bin"
```

Comment:
Could you avoid changing the alignment here? It makes the diff bigger for no benefit it seems (it's even less readable imho).

Reply:
The file uses alignment, so this was aligned to match rest of the file. I think it's better to match the file style.

Reply:
Fair enough for this case. Although I really don't like because it changes unrelated lines for no need and often doesn't help readability.